requirements-dev.txt

```diff
@@ -10,4 +10,5 @@ scanf>=1.4.1
 pytest
 pytest-cov
 coverage
-flake8
\ No newline at end of file
+flake8
+tox
```

Comment:
A quick grep seems to show that both `requirements.txt` and `requirements-dev.txt` are almost completely unreferenced now.

The only place either is mentioned is in old release notes and in `README.rst`, which suggests `pip install -r requirements-dev.txt` as part of generating the sphinx docs.

Would it make sense to remove that reference from `README.rst` and then remove the `requirements` files completely? That way we could avoid duplication / opportunity for mismatch between them and `setup.py`.